pelican-gfm/Readme.md

```diff
@@ -0,0 +1,22 @@
+pelican-gfm
```

Comment:
From my understanding, this plugin will only work on Debian systems.
Better mention this OS limitation in the README